### ISSUE TITLE
Enable SSL encryption for HDX web sites

### ISSUE THREAD

**Issue by teodorescuserban** [OCHA-DAP/hdx-infrastructure#36]
When the new certificate arrives, I will manage the transition to ssl encryption for all sites we have.

**Comment by davidmegginson** [comment]
Blocked by #36.

**Background:** (not needed by Serban :-) ).  This issue is critical because without it, passwords and other confidential information will be transmitted in the clear over the Internet.

**Comment by teodorescuserban** [comment]
And let me add here that our infrastructure is immune to OpenSSL Heartbleed Bug (it has been vaccinated).

**Comment by teodorescuserban** [comment]
Blocked by certificate purchase.

**Comment by teodorescuserban** [comment]
Assigned to @cjhendrix to confirm purchase.

**Comment by teodorescuserban** [comment]
Domain verification for certificates then we are good to go.

**Comment by teodorescuserban** [comment]
Certificates have been downloaded and sent to our provider to install them.
ETA: 2-3 days top.

**Comment by amcguire62** [comment]
Can i check was this done?

**Comment by teodorescuserban** [comment]
Further messages between me and our provider revealed that we need to change the external ip address. This may lead to temporary or partial downtime and it wad decided to postpone it at least after Kenya. I didnt try again to switch to ssl afterwards with all changes and demos...

**Comment by teodorescuserban** [comment]
Update:
1. production ckan and cps are on ssl now (ckan for two days, cps for a day).
2. staging ckan and cps are on ssl; staging third vhost on ssl as well.
3. dev ckan and cps - well it doesnt matter - vpn only.

only blog remains.

**Comment by luiscape** [comment]
Working on the blog. I'm having to take care of other issues in the pipeline. I'm hoping to complete this issue this week.

**Comment by teodorescuserban** [comment]
any news, @luiscape ?

**Comment by teodorescuserban** [comment]
closing.
only https://github.com/OCHA-DAP/DAP-Blog/issues/94 remains.
